packages/service-library/src/servicelib/openapi.py

```diff
@@ -37,23 +36,20 @@ def get_base_path(specs: OpenApiSpec) ->str:
     return '/v' + specs.info.version.split('.')[0]
 
 
+# TODO: _load_from_* is also found in jsonshema_specs
```

Comment:
This will come in a [different PR](https://github.com/pcrespov/osparc-simcore/commit/02f279415bdbfc8e1160c7f44867bc4743087d10) ... I thought it was too much change for a single PR